commands/setUser.js

```diff
@@ -84,21 +84,29 @@ Use \`/iam\` to verify your email address.`,
 
   await db.close();
 
+  let count = 0;
   rows.forEach(async(user) => {
     let id = user['userid'];
     let server_member = await server.members.fetch(id);
 
-    if (server_member.roles.cache.some((role) => role.name === officer_role.name)) {
-      await server_member.roles.remove(officer_role.id);
-      await server_member.roles.add(alumni_officer_role);
+    if(server_member) {
+      try {
+        if (server_member.roles.cache.some((role) => role.name === officer_role.name)) {
+          await server_member.roles.remove(officer_role.id);
```

Comment:
If the user also has committee officer roles, should we remove these roles/change them to emeritus as well? @advaithg

Reply:
Table to @danieljaffe since he wrote the ticket!! both seem like they would work for me!

Reply:
going to make an executive decision here and say removing them is fine for now, and we can modify it later if there's any objections!